pepper/cli.py

```diff
@@ -322,6 +326,21 @@ def parse_cmd(self):
                 logger.error("Invalid JSON given.")
                 raise SystemExit(1)
 
+        if self.options.json_file:
+            try:
+                with open(self.options.json_file, 'r') as json_content:
+                    try:
+                        return json.load(json_content)
+                    except ValueError:
+                        logger.error("Invalid JSON given.")
+                        raise SystemExit(1)
+            except Exception as e:
+                logger.error(
+                    'Impossible to process with the file : {0}, {1}'.format(
```

Comment:
I would raise with something like "Cannot read file", because in this place exception would most likely mean the file doesn't exist on the given path or you don't have permissions to access it.